napari/conftest.py

```diff
@@ -434,7 +434,20 @@ def _mock_app():
             Application.destroy('test_app')
 
 
-def _get_calling_place(depth=1):
+def _get_calling_stack():  # pragma: no cover
+    stack = []
+    i = 2
+    while True:
+        try:
+            frame = sys._getframe(i)
+        except ValueError:
+            break
+        stack.append(f"{frame.f_code.co_filename}:{frame.f_lineno}")
+        i += 1
```

Comment:
I don't love `while True` loops. Alternative suggestion? (I don't know whether the recursion depth is indeed only for recursion or it's a general stack depth limit. Either way this should be safe enough I think.)

```suggestion
    for i in range(2, sys.getrecursionlimit()):
        try:
            frame = sys._getframe(i)
        except ValueError:
            break
        stack.append(f"{frame.f_code.co_filename}:{frame.f_lineno}")
```